MVVM/ViewModel/SafeMode_UIOverlay.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Godot;
+using RoverControlApp.Core;
+using RoverControlApp.MVVM.Model;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace RoverControlApp.MVVM.ViewModel;
+
+public partial class SafeMode_UIOverlay : UIOverlay
+{
+	[Export]
+	PanelContainer _panelContainer;
+
+	private int _inputMode;
+
+	private static float _speedLimit => LocalSettings.Singleton.SpeedLimiter.MaxSpeed;
+
+	public override Dictionary<int, Setting> Presets { get; } = new() 
+	{
+		{ 0, new(Colors.Blue, Colors.LightBlue, $"SpeedLimiter: ON {_speedLimit:P0}", "SpeedLimiter: ") },
+		{ 1, new(Colors.DarkRed, Colors.Orange, "SpeedLimiter: OFF", "SpeedLimiter: ") }
+
+	};
+
+	public Task ControlModeChangedSubscriber(MqttClasses.ControlMode newMode)
+	{
+		_inputMode = (int)newMode;
+		CallDeferred(MethodName.UpdateSafeModeIndicatator);
+		return Task.CompletedTask;
+	}
+
+	public override void _Ready()
+	{
+		base._Ready();
+		LocalSettings.Singleton.Connect(LocalSettings.SignalName.PropagatedPropertyChanged,
+			Callable.From<StringName, StringName, Variant, Variant>(OnSettingsPropertyChanged));
+		UpdateDictionary();
+	}
+
+	void OnSettingsPropertyChanged(StringName category, StringName name, Variant oldValue, Variant newValue)
+	{
+		if (category != nameof(LocalSettings.SpeedLimiter))
+			return;
+		UpdateDictionary();
+
+		UpdateSafeModeIndicatator();
+	}
+
+	void UpdateDictionary()
+	{
+		Presets[0] = new(Colors.Blue, Colors.LightBlue, $"SpeedLimiter: ON {_speedLimit:P0}", "SpeedLimiter: ");
+	}
+
+	void UpdateSafeModeIndicatator()
+	{
+		if (_inputMode != 1)
+		{
+			_panelContainer.Visible = false; 
```

Comment:
Using `this.Visible` is sufficient here.
Using `_panelContainer.Visible` is still working fine but it's longer way around.

Reply:
Adressed in [5ed280b](https://github.com/wisniax/RoverControlApp/pull/36/commits/5ed280b9da4555ee40b428e3d684a0ecffc34549)